src/Compilers/CSharp/Portable/Lowering/Instrumentation/DynamicAnalysisInjector.cs

```diff
@@ -129,6 +129,31 @@ private static bool IsExcludedFromCodeCoverage(MethodSymbol method)
             return false;
         }
 
+        private BoundExpression GetFileIndices()
```

Comment:
Fixed - This method no longer exists but I have made the return type for the corresponding method most specific.